plotter/histogram.go

```diff
@@ -32,6 +32,9 @@ type Histogram struct {
 	// LineStyle is the style of the outline of each
 	// bar of the histogram.
 	draw.LineStyle
+
+	// LogY enables support for histograms displayed with a Y log-scale.
```

Comment:
This comment is not quite right; `// LogY allows rendering with a log-scaled Y axis.` (or something similar), but also a brief explanation of what it is doing to make this possible.

Reply:
done.

Reply:
It would still be nice to have an explanation of what it does.

Reply:
tentatively: aca1101